tests/common.cpp

```diff
@@ -0,0 +1,229 @@
+#include "common.h"
+#include <cmath>
+#include <cstdint>
+
+namespace SSE2NEON
+{
+int32_t nan = ~0;
+int64_t nan64 = ~0;
+
+result_t validate128(__m128i a, __m128i b)
```

Comment:
I am not sure about returning the `result_t` is a good idea since there is no way that it would return `TEST_UNIMPL`.

Reply:
I changed the type to `result_t` since we could return the `validateXX` function directly in test functions.